### PR TITLE
SLO type name change + new default view functionality

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
Updates event-based SLOs to the new name: metric-based SLO

### Motivation
UI change + PM request

### Preview link

There should be no mention of event-based here:
- https://docs-staging.datadoghq.com/kaylyn/sli/monitors/service_level_objectives/event/
- https://docs-staging.datadoghq.com/kaylyn/sli/monitors/service_level_objectives/

New saved views section:
- https://docs-staging.datadoghq.com/kaylyn/sli/monitors/service_level_objectives/#view-your-slos

### CODE REVIEW

File: content/en/monitors/service_level_objectives/metric.md [hunk 1]
```diff
@@ -10,12 +12,12 @@ further_reading:
 
 ## Overview
 
-Event-based SLOs are useful for a count-based stream of data where you are differentiating good and bad events. 
-There is one type of event-based SLO - a metric query SLO. A metric query uses the sum of the good events divided by the sum of total events over time to calculate a Service Level Indicator (or SLI).
+Metric-based SLOs are useful for a count-based stream of data where you are differentiating good and bad events. 
+There is one type of metric-based SLO - a metric query SLO. A metric query uses the sum of the good events divided by the sum of total events over time to calculate a Service Level Indicator (or SLI).
```
Comment (meghanelizabeth) [inline content/en/monitors/service_level_objectives/metric.md:16]: Let's reword: 

> There is one type of metric-based SLO - a metric query SLO

 I don't think it's necessary with the wording change.

File: content/en/monitors/service_level_objectives/_index.md [hunk 2]
```diff
@@ -85,6 +83,50 @@ This can result in the overall uptime being lower than the average of the indivi
...
+
+The default view includes:
+
+- An empty search query
+- A list of all defined SLOs in your organization 
+- A list of available facets in left side facet list
+
+### Saved Views
+
+Saved views allow you to save and share across your organization search customizations in the SLO list view for SLOs that are most relevant for you including:
```
Comment (meghanelizabeth) [inline content/en/monitors/service_level_objectives/_index.md:98]: Saved views allow you to save and share customized searches in the SLO list view for SLOs that are most relevant for you including:

Sound better?

File: content/en/monitors/service_level_objectives/_index.md [hunk 3]
```diff
@@ -85,6 +83,50 @@ This can result in the overall uptime being lower than the average of the indivi
...
+Star saved views to mark them as favorites. Starred saved views are available directly from the main navigation menu.
+
+#### Share a Saved View
+
+Copy-paste a saved view short-link to share it with your teammates.
+
+#### Manage Saved Views
+
+To create a new saved view, click the *Save as* button at the top of the screen. Give it a name and click *Save*.
+
```
Comment (meghanelizabeth) [inline content/en/monitors/service_level_objectives/_index.md:127]: Add: 

Once you are using a saved view, you can update it by selecting that view, modifying the query, and clicking Update.

File: content/en/monitors/service_level_objectives/metric.md [hunk 4]
```diff
@@ -10,12 +12,12 @@ further_reading:
...
 
 ## Setup
 
-On the [SLO status page][1], select **New SLO +**. Then select [**Event**][2].
+On the [SLO status page][1], select **New SLO +**. Then select [**Metric**][2].
 
 ### Configuration
```
Comment (phillip-dd) [inline content/en/monitors/service_level_objectives/metric.md:23]: Can comment on it, but the last line in the file is `[2]: https://app.datadoghq.com/slo/new/event`, I assume we'll update that link and redirect, but it should probably be `[2]: https://app.datadoghq.com/slo/new/metric`

File: content/en/monitors/service_level_objectives/_index.md (hunk 2 above)
Comment (athap) [reply content/en/monitors/service_level_objectives/_index.md:98]: I think so

File: content/en/monitors/service_level_objectives/monitor.md [hunk 5]
```diff
@@ -69,4 +69,4 @@ Confirm you are using the preferred SLI type for your use case. Datadog supports
 [2]: https://app.datadoghq.com/slo
 [3]: https://app.datadoghq.com/slo/new/monitor
 [4]: https://app.datadoghq.com/monitors#create
-[5]: /monitors/service_level_objectives/event/
+[5]: /monitors/service_level_objectives/metric/
```
Comment (athap) [inline content/en/monitors/service_level_objectives/monitor.md:72]: This doesn't work currently but I am guess this is being worked on?

File: content/en/monitors/service_level_objectives/metric.md (hunk 4 above)
Comment (meghanelizabeth) [reply content/en/monitors/service_level_objectives/metric.md:23]: nice catch

File: content/en/monitors/service_level_objectives/monitor.md (hunk 5 above)
Comment (meghanelizabeth) [reply content/en/monitors/service_level_objectives/monitor.md:72]: Yeah a part of the docs update

File: content/en/monitors/service_level_objectives/_index.md [hunk 6]
```diff
@@ -85,6 +83,50 @@ This can result in the overall uptime being lower than the average of the indivi
...
+- A search query
+- A selected subset of facets
+
+After you query for a subset of SLOs on the list view, you can now add that query as a saved view. 
+
+#### Add a saved view
+
+To add a saved view:
+1. Query for your SLOs.
```
Comment (l0k0ms) [inline content/en/monitors/service_level_objectives/_index.md:108]:
```suggestion
To add a saved view:

1. Query for your SLOs. 
```

File: content/en/monitors/service_level_objectives/_index.md [hunk 7]
```diff
@@ -85,6 +83,50 @@ This can result in the overall uptime being lower than the average of the indivi
...
+
+Star saved views to mark them as favorites. Starred saved views are available directly from the main navigation menu.
+
+#### Share a Saved View
+
+Copy-paste a saved view short-link to share it with your teammates.
+
+#### Manage Saved Views
+
+To create a new saved view, click the *Save as* button at the top of the screen. Give it a name and click *Save*.
```
Comment (apigirl) [inline content/en/monitors/service_level_objectives/_index.md:126]:
```suggestion
To create a new saved view, click the *Save as* button at the top of the screen. Give it a name and click *Save*. Once you are using a saved view, you can update it by selecting that view, modifying the query, and clicking *Update*.
```

**Pull request merged**